### DISCUSSION TITLE
添加插件功能、完善命令行

### DISCUSSION THREAD

**Discussion by pyly-pyly** [Meloong-Git/PCL#5269] [讨论]
### 添加插件功能
插件开发者可以根据文档来开发PCL插件，用户可以自由选择插件来给PCL添加功能

### 完善命令行
命令行支持更多功能，包括启动游戏、查看游戏信息等

## 其它内容

（如果这个被选上了，请标明**作者PYLY**，因为我GitHub名称本来是要用PYLY的，结果重名，所以只能用pyly-pyly）

**Comment by hejiehao** [comment]
> 添加插件功能

- https://github.com/Hex-Dragon/PCL2/issues/441

> 完善命令行

- https://github.com/Hex-Dragon/PCL2/issues/2214

**Comment by allMagicNB** [comment]
……首先，没理由要标你。连社区贡献者都没加自己署名，你提个建议还要写你名了？

**Reply by Deep-Dark-Forest** [reply]
他的意思可能是在如果加进去了，更新日志里写上他要求的名字而不是他的gh名字
